### ISSUE TITLE
finalize detection webhook JSON shapes

### ISSUE THREAD

**Issue by iameli** [livepeer/go-livepeer#1917]
In an effort to maximize the usability of the scene classification API, as well as keep things as consistent as possible between go-livepeer and livepeer-com, the following is proposed as the shape for the scene detection webhook workflow, based on discussion and consensus with @yondonfu @ericxtang @victorges. Some basic objectives:

* Replace opaque integer IDs with human-readable strings
* Replace fraction types with floating-point numbers; exact precision isn't as necessary when we're talking about being 96.1391296% certain that there is soccer on the screen
* Keep the API shape as consistent as possible; when possible the same scene classification concepts should be referenced in the same way

## Incoming auth webhook response from API server:
```json5
{
    // ... other fields here, "profiles" and "manifestID" and whatnot ...

    detection: {
        frequency: 1,
        sampleRate: 100,
        sceneClassification: [
            {name: "soccer"},
            {name: "adult"}
        ],
        
        // someday, we'll have other detection-related fields here, like:
        objectDetection: [
            {name: "soccer ball"}
        ]
    }
}
```

## Outgoing classification message to `-detectionWebhookURL`
```json5
{
    manifestID: "ABC123",
    seqNo: 100,
    sceneClassification: [
        {name: "soccer", probability: 0.5},
        {name: "adult", probability: 0.01}
    ]
    
    // and some
    objectDetection: [
        {name: "soccer ball", coords: [1,2,3,4]}
    ]
}
```

**Comment by iameli** [comment]
I don't have a strong opinion about whether the gRPC protocol should be updated to be more in line with this structure, but it seems worth considering at least the replacement of integer ids with string enums for model identification

**Comment by yondonfu** [comment]
Closed by #1922
